src/traveler.js

```diff
@@ -57,7 +57,7 @@ class Traveler {
       let tripCost;
       destinations.forEach(destination => {
         if (trip.destinationID === destination.id) {
-          tripCost = (trip.travelers * ((trip.duration * destination.estimatedLodgingCostPerDay) + destination.estimatedFlightCostPerPerson))
+          tripCost = (1.1 * (trip.travelers * ((trip.duration * destination.estimatedLodgingCostPerDay) + destination.estimatedFlightCostPerPerson)));
```

Comment:
Math looks right! You could set them to variables like: 
 estimatedLodgingCostForTrip = (duration * estimatedLodgingCostPerDay)
estimatedTotalCostForIndividualTrip = (estimatedLodgingCostForTrip + estimatedFlightCostPerPerson)
estimatedTotalCostForAllTrip = (estimatedTotalCostForIndividualTrip * travelers)
tripCost = (estimatedTotalCostForAllTrip * 1.1)
But that's more developer's choice.